BizHawk.Common/MutableIntRange.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+
+namespace BizHawk.Common
+{
+	public class MutableIntRange
+	{
+		public int Min;
```

Comment:
My one nag is that these should be properties instead of fields

Reply:
Fixed